tests/integ/modin/series/test_unique.py

```diff
@@ -26,7 +26,7 @@ def _make_nan_interleaved_float_series():
     [12.0, 11.999999, 11.999999],
     ["A", "A", "C", "C", "A"],
     [None, "A", None, "B"],
```

Comment:
I think this actually on purpose testing these edge-case values because of the uint64. 

Is there a ticket to fix the underlying cause? 

If this is a blocker for CI, maybe we can mark it as XFAIL together with a ticket to fix this eventually? 

I am not sure it's a good idea to "tune" the tests to work. E.g., a follow up question is - do you think 2**34 would work? That's an uint64 number but not a int64 number. Here, we're explicitly testing with numbers out of int64 range.

Reply:
I agree with @sfc-gh-lspiegelberg . First to ask what is the root cause?